docs/tutorials/05-model-monitoring.ipynb

```diff
@@ -93,21 +98,24 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from src.model_monitoring_utils import enable_model_monitoring\n",
-    "\n",
-    "# If this project was running with MM enabled pre-1.8.0, disable the old model monitoring to update configurations\n",
-    "project.disable_model_monitoring(delete_stream_function=True)\n",
-    "\n",
-    "enable_model_monitoring(project=project, base_period=2)"
+    "from src.model_monitoring_utils import enable_model_monitoring"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    ":::{Admonition} Note\n",
+    "If this project was running with model monitoring enabled in a pre-1.8.0 version, disable the old model monitoring to update configurations:\n",
+    "```\n",
+    "project.disable_model_monitoring(delete_stream_function=True)\n",
+    "enable_model_monitoring(project=project, base_period=12)\n",
+    "```\n",
+    ":::\n",
     "### Log the model artifact\n",
     "\n",
-    "See the parameter descriptions in [`log_model`](https://docs.mlrun.org/en/stable/api/mlrun.projects/index.html#mlrun.projects.MlrunProject.log_model). "
+    "This tutorial uses an existing MLRun dataset for training. You just need to specify it. \n",
+    "Then you can log the model. See the parameter descriptions in [`log_model`](https://docs.mlrun.org/en/stable/api/mlrun.projects/index.html#mlrun.projects.MlrunProject.log_model). "
```

Comment:
this is good but after this cell we are jumping directly to `Use the [v2_model_server serving](https://www.mlrun.org/hub/functions/master/v2-model-server/) function from the MLRun function hub.` without explaining why we need it, what is v2 model server, and what is the mlrun hub. Again, according to Nir's review, we have to add more context here and "tell a story", something like:
"Once the model is logged, the next step is to create the serving function (maybe a link to a docs page explaining what is a serving function) that will be used to invoke that model. We will use a pre-defined function called `v2_model_server` that can be found in MLRun HUB (link to the hub func)... "